mobile/src/main/java/org/openhab/habdroid/ui/OpenHABWidgetAdapter.java

```diff
@@ -707,19 +707,19 @@ public void onClick(DialogInterface dialog, int which) {
                             final NumberPicker numberPicker = (NumberPicker) dialogView.findViewById(R.id.numberpicker);
                             int minValue = (int) openHABWidget.getMinValue();
                             int maxValue = (int) openHABWidget.getMaxValue();
-                            int stepsize = (int) openHABWidget.getStep();
-                            final int stepsizeSave;
+                            int stepSize = (int) openHABWidget.getStep();
+                            final int stepSizeSave;
```

Comment:
Maybe there is a better way.

Reply:
What were you trying to accomplish?

Reply:
Changing stepsize even when its final.

Reply:
```
                            int minValue = (int) openHABWidget.getMinValue();
                            int maxValue = (int) openHABWidget.getMaxValue();
                            final int stepSize;
                            if(minValue == maxValue) {
                                stepSize = 1;
                            } else {
                                stepSize = (int) openHABWidget.getStep();
                            }

                            numberPicker.setMaxValue(maxValue/stepSize);
                            numberPicker.setMinValue(minValue);
                            NumberPicker.Formatter formatter = new NumberPicker.Formatter() {
                                @Override
                                public String format(int value) {
                                    int temp = value * stepSize;
                                    return "" + temp;
                                }
                            };
```

?

Reply:
stepSize must be declared final, but I want to set it to 1 in case minValue=maxValue

Reply:
yeah, that is what I do above in that snippet.  declared final, if min==max its 1, otherwise it gets step from the widget.

Reply:
Thats better than mine. Can you commit it?

Reply:
done.